Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2655,21 +2594,22 @@ extension BrowserViewController: JSPromptAlertControllerDelegate {
 
 extension BrowserViewController: TopTabsDelegate {
     func topTabsDidPressTabs() {
-        leaveOverlayMode(didCancel: true)
+        // Technically is not changing tabs but is loosing focus on urlbar
+        overlayManager.switchTab(didCancel: true)
```

Comment:
I am trying to understand what this does. This isn't the same as leaving overlay mode? Why is there `switchTab` in the function name?

Reply:
The decision is related to:
> Update protocol function names to be directly related to actions that involved enter/leave overlay mode. At the moment the logic in this function is pretty simple most of them just call enterOverlayMode or leaveOverlayMode but as we build the new behavior we could logic easily

The reasoning behind this is to have a clear idea of what is the action that is toggling the keyboard and avoid the over use of enter/leaveOverlayMode